src/utils/fundingCycle.ts

```diff
@@ -85,7 +86,10 @@ export const hasFundingTarget = (
 ) => fundingCycle.target.lt(constants.MaxUint256)
 
 export const hasFundingDuration = (
-  fundingCycle: Pick<V1FundingCycle | EditingV1FundingCycle, 'duration'>,
+  fundingCycle: Pick<
+    V1FundingCycle | EditingV1FundingCycle | V2FundingCycleData,
```

Comment:
**issue**; lets make a new v2 file for this. I don't think we should be mixing v1 and v2 utils.

Don't worry about moving this file to utils/v1/ as I'll handle it in https://github.com/jbx-protocol/juice-interface/pull/569

Reply:
@aeolianeth yep moved into separate file `fundingCycleV2.ts`